basic-android-kotlin-compose-training-mars-photos-starter/app/src/main/java/com/example/marsphotos/ui/screens/realEstateScreen/RealEstateScreen.kt

```diff
@@ -13,12 +13,13 @@ import com.example.marsphotos.ui.screens.realEstateScreen.components.RealEstateC
 fun RealEstateScreen(
     realEstateUiState: RealEstateUiState,
     modifier: Modifier = Modifier,
-    navController: NavHostController
+    navController: NavHostController,
+    viewModel: RealEstateViewModel,
+    selectedPosition: Int
 ) {
     when (realEstateUiState) {
         is RealEstateUiState.Loading -> LoadingScreen(modifier = modifier.fillMaxSize())
-        is RealEstateUiState.Success -> RealEstateColumnScreen(realEstateUiState.realEstates, modifier, navController = navController)
+        is RealEstateUiState.Success -> RealEstateColumnScreen(realEstateUiState.realEstates, modifier, navController = navController, viewModel, selectedPosition)
```

Comment:
Quite a long line of code here. Couldn't we make it look better? Consider the question with named parameters below.